apps/explorer/src/ui/Amount.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Heading } from '~/ui/Heading';
+import { Text } from '~/ui/Text';
+
+export type AmountProps = { amount: number; coinSymbol?: string };
+
+export function Amount({ amount, coinSymbol }: AmountProps) {
+    return (
+        <div className="flex flex-row items-end gap-1 text-sui-grey-100 ml-6">
+            <Heading as="h4" variant="heading4">
```

Comment:
This shouldn't be an h4, that has a meaning for accessibility that this does not have.

Reply:
done